docs/Getting Started.md

```diff
@@ -90,7 +92,17 @@ Cloudflow offers several sbt plugins that abstract quite a bit of boilerplates n
 
 > **Note:** You can use multiple plugins to develop an application that uses multiple runtimes (Akka, Spark, Flink etc.). For simplicity of this example we will be using only one.
 
-The above build script is standard Scala sbt - the only difference is the plugin which we provide as part of Cloudflow.
+The above build script is standard Scala sbt - the only difference is the plugin which we provide as part of Cloudflow. And here's how we specify the name and version of the plugin to use:
+
+**project/cloudflow-plugins.sbt**
+
+```
+// Resolver for the cloudflow-sbt plugin
+//
+resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/"
+
+addSbtPlugin("com.lightbend.cloudflow" % "sbt-cloudflow" % "1.3.0-M1")
```

Comment:
Shouldn't this be M5?

Reply:
ok - will change .. finally we should have `1.3.0`